datadog/api/api_client.py

```diff
@@ -135,6 +135,11 @@ def submit(cls, method, path, api_version=None, body=None, attach_host_name=Fals
             if 'tags' in params and isinstance(params['tags'], list):
                 params['tags'] = ','.join(params['tags'])
 
+            # If defined, make sure monitor_ids are defined as a comma-separated string
+            if 'monitor_ids' in params and isinstance(params['monitor_ids'], list):
+                mapped_monitor_ids = [str(i) for i in params['monitor_ids']]
+                params['monitor_ids'] = ','.join(mapped_monitor_ids)
```

Comment:
```suggestion
                params['monitor_ids'] = ','.join(str(i) for i in params['monitor_ids'])
```
or `mapped_monitor_ids = (str(i) for i in params['monitor_ids'])` as there is no need to create a new list.